jekyll/_cci2/set-up-multiple-configuration-files-for-a-project.adoc

```diff
@@ -54,14 +56,29 @@ Next, create a trigger for your pipeline.
 
 . Following from the previous step, select **Triggers** in the sidebar
 +
-. Select btn:[Add Trigger]
-. Select **GitHub App** in the dropdown menu
-. Select btn:[Next]
+. Select btn:[Add Trigger].
+. Select **GitHub App** in the dropdown menu.
+. Select btn:[Next].
 . Complete the required form fields:
-** Give your trigger a descriptive name
-** Select your project repository from the dropdown
-** Choose your new pipeline from the "Choose config to run" menu
+** Give your trigger a descriptive name.
+** Select your project repository from the dropdown.
+** Choose your new pipeline from the "Choose config to run" menu.
+** Choose a trigger event options from the "Run on" menu.
++
+[TIP]
+====
+Using multiple pipelines and triggers, you can build  different pipelines for different events, For example, you could have:
+
+- A `build-test` pipeline that runs on every push. Select "All pushes".
+- A `e2e-tests` pipeline, longer and more costly, that runs once when the PR is taken out of draft. Select "PR marked ready for review".
+- A `teardown-env` pipeline that runs when a PR is merged. Select "PR merged".
+
+For a full list, see the xref:github-trigger-event-options#[GitHub trigger event options] page.
+====
```

Comment:
How about moving this outside of a "tip" and above "prerequisites"? I think it describes the value of using multiple pipelines, so it can entice a reader to continue reading

Reply:
good idea!